src/components/admin/Button/Button.tailwind.ts

```diff
@@ -6,6 +6,8 @@ export const secondary =
   "border-gray-300 bg-white text-gray-700 hover:bg-gray-50 focus:ring-indigo-500";
 export const danger =
   "border-transparent text-white bg-warning-red focus:ring-indigo-500";
+export const secondaryBorderless =
+  "border-none shadow-none bg-white text-gray-700 hover:bg-gray-50 focus:ring-indigo-500";
```

Comment:
We're not constrained to have a single `variant` prop, we can also have a `border` one so we can do combinations if that makes sense. We might have similar issues with this button as the one for attendee.

@denisdimitrov Please check this comment and how it was resolved: https://github.com/sparkletown/sparkle/pull/2905/files#r823519370

And here it is how it's being used https://github.com/sparkletown/sparkle/pull/2905#discussion_r823546585